demo/ThemeTransform.html

```diff
@@ -51,6 +51,7 @@
   <body>
     <div id='theme_demo_tools'>
       <control-panel></control-panel>
+      <input type='checkbox' id='collapse_tables_checkbox'> <label for=collapse_tables_checkbox>Collapse tables</label>
```

Comment:
A checkbox doesn't seem quite right here since there's no undo-ing the transform but I think this is a known issue.